ui/selectors/selectors.js

```diff
@@ -574,7 +574,7 @@ export function getSuggestedNfts(state) {
     getUnapprovedConfirmations(state)?.filter(({ requestData, type }) => {
       return (
         type === ApprovalType.WatchAsset &&
-        [ERC721, ERC1155].includes(requestData?.asset?.standard)
+        requestData?.asset?.tokenId !== undefined
```

Comment:
We should consider it a suggestedNFT if `tokenId` is defined not based on whether `standard` is defined.